aws_lambda_powertools/utilities/idempotency/base.py

```diff
@@ -51,6 +57,7 @@ def __init__(
         function_payload: Any,
         config: IdempotencyConfig,
         persistence_store: BasePersistenceLayer,
+        output_serializer: Optional[BaseDictSerializer] = None,
```

Comment:
I think you meant:

```suggestion
        output_serializer: BaseDictSerializer = NoOpSerializer(),
```

Reply:
Once I try this change I receive the 'make pr' error
B008 Do not perform function call `NoOpSerializer` in argument defaults.

1. Should I still perform this change?
2. If I should, how can I bypass/ignore this error?

Reply:
Sorry, I reconsidered. Your code looks good, leave it as it is!